app/js/components/questions/new/Index.react.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import * as QuestionActions from '../../../actions/QuestionActions.js';
+import NewQuestionPage from './NewQuestionPage.react';
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import toastr from 'toastr';
+
+class NewQuestion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Perhaps this some of this initial state are no longer needed as they are now being handled in the reducers :)